website/docs/r/private_dns_glb_pool.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "ibm"
+page_title: "IBM : dns_glb_pool"
+sidebar_current: "docs-ibm-resource-dns-glb-pool"
+description: |-
+  Manages IBM Private DNS glb pool.
+---
+
+# ibm_dns_glb_pool
+
+Provides a private dns glb pool resource. This allows dns glb pool to be created,updated and deleted.
+
+## Example Usage
+
+```hcl
+
+resource "ibm_dns_glb_pool" "test-pdns-glb-pool-nw" {
+    instance_id = ibm_resource_instance.test-pdns-instance.guid
+    vpc_crn = ibm_is_vpc.test_pdns_vpc.crn
+    type = "vpc"
+    description = "new nimi pool"
+    enabled=true
+    healthy_origins_threshold=1
+    origins {
+	    name    = "example-1"
+     	address = "www.google.com"
+	    enabled = true
+	    description="origin pool"
+    }
+    monitor="7dd6841c-264e-11ea-88df-062967242a6a"
+    notification_channel="https://mywebsite.com/dns/webhook"
+    healthcheck_region="us-south"
+    healthcheck_subnets=["0716-a4c0c123-594c-4ef4-ace3-a08858540b5e"]
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `instance_id` - (Required, string) The id of the private DNS on which zone has to be created.
+- `name` - (Required, string) Name of the load balancer pool.
+- `description` - (Optional,string) Descriptive text of the load balancer pool.
+- `enabled` - (Optional,bool) Whether the load balancer pool is enabled.
+- `healthy_origins_threshold` - (Optional,string) The minimum number of origins that must be healthy for this pool to serve traffic. If the number of healthy origins falls below this number, the pool will be marked unhealthy and we will failover to the next available pool.
```

Comment:
schema and docs has conflicts,
Please make sure attribute types are same in schema and in docs
Required|Optional mismatch..schema says required doc says option for origins. enabled..
Please cross check all the attributes

Reply:
resolved